nepalingo-web/src/pages/Home/Home.tsx

```diff
@@ -42,7 +42,7 @@ const Home: React.FC = () => {
       </div>
       <div className="mb-10">
```

Comment:
In Home - instead of this button, we should use the DailyQuiz component that you made